GPU/GPUTracking/TPCClusterFinder/GPUTPCClusterFinder.h

```diff
@@ -19,6 +19,10 @@
 #include "GPUProcessor.h"
 #include "GPUDataTypes.h"
 #include "CfFragment.h"
+#include "ML/OrtInterface.h"
```

Comment:
this class goes to CUDA/OpenCL/ROCm. It must not contain any OrtInterface stuff nor any std::... types.
Also it will go to the GPUConstantCache, so it must be small.
A workaround for now would be:
- Create a GPUTPCClusterFinderNNInternals class and header file.
- move all the stuff in there.
- forward-declare that class in here, and store only a pointer to that class in here.